src/turnkeyml/common/filesystem.py

```diff
@@ -365,6 +365,12 @@ class Keys:
     # Prefix for reporting the execution status of a stage
     # In the report this will look like stage_status:STAGE_NAME
     STAGE_STATUS = "stage_status"
+    # Parent key that holds all of the arguments to turnkey's
+    # evaluate_file() API
+    EVALUATION_ARGS = "args"
```

Comment:
It is really nice to have "args" as part of the report, but it is also very long.
Args also starts with "a", so this is the fist thing that one sees when opening the report and there is a lot of scrolling that needs to be done before getting to the rest of the content.

Should we call this `tkml_args` or `turnkey_args` to improve the navigation through the raw csv file?
```suggestion
    EVALUATION_ARGS = "turnkey_args"
```

Reply:
yeah I like this suggestion. "args" was a little ambiguous anyways.